dynalab_cli/test.py

```diff
@@ -64,132 +83,154 @@ def run_command(self):
-                universal_newlines=True,
-            )
-            if process.returncode != 0:
-                raise RuntimeError(f"Exception in torchserve archive {process.stderr}")
-
-            # pull docker
-            lib_dir = os.path.join(
-                os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
-                "dynalab",
-                "dockerfiles",
-            )
-            docker_path = os.path.join(tmp_dir, "Dockerfile")
-            # TODO: pull the files from dynalab repo once public
-            shutil.copyfile(os.path.join(lib_dir, "Dockerfile.dev"), docker_path)
-            shutil.copyfile(
-                os.path.join(lib_dir, "dev-docker-entrypoint.sh"),
-                os.path.join(tmp_dir, "dev-docker-entrypoint.sh"),
+            self.run_docker_test(config)
+
+    def run_docker_test(self, config):
+        tmp_dir = os.path.join(self.config_handler.config_dir, "tmp")
+        os.makedirs(tmp_dir, exist_ok=True)
+        # tarball everything
+        print("Tarballing the project directory...")
+        exclude_list_file = os.path.join(tmp_dir, "exclude.txt")
+        self.config_handler.write_exclude_filelist(
+            exclude_list_file, self.args.name, exclude_model=True
+        )
+        process = subprocess.run(
+            [
+                "tar",
+                f"--exclude-from={exclude_list_file}",
+                "-czf",
+                os.path.join(tmp_dir, f"{self.args.name}.tar.gz"),
+                ".",
+            ],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+        )
+        if process.returncode != 0:
+            raise RuntimeError(
+                f"Exception in tarballing the project directory {process.stderr}"
             )
 
-            # build docker
-            repository_name = self.args.name.lower()
-            print("Building docker image...")
-            docker_build_args = [
-                "--build-arg",
-                f"add_dir={tmp_dir}",
-                "--build-arg",
-                f"model_name={self.args.name}",
-                "--build-arg",
-                f"requirements={str(config['requirements'])}",
-                "--build-arg",
-                f"setup={str(config['setup'])}",
-            ]
-            docker_build_command = [
-                "docker",
-                "build",
-                "--network",
-                "host",
-                "-t",
-                repository_name,
-                "-f",
-                docker_path,
-                ".",
-            ] + docker_build_args
-
-            subprocess.run(docker_build_command)
-            # NOTE: cpu and memory limit are specific to ml.m5.xlarge
-            process = subprocess.run(
-                f"docker run --network none --cpus 4 --memory 16G {repository_name}",
-                shell=True,
-                stderr=subprocess.PIPE,
```

Comment:
The reason we moved all torchserve logs into stderr, and print out stdout here is that, during integrated test, we want people to see the test data and test response on the console (i.e. the printouts in the docker entrypoint file), but redirect all the torchserve logs into the file for later inspection so it doesn't crowd the console. 

Can we resume to the previous design please?